### PR TITLE
[panOS] Add Security Rule does not take list as paramter

### PR DESCRIPTION
**Pull request by samuelFain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [XSUP-24694](https://jira-hq.paloaltonetworks.local/browse/XSUP-24694)

## Description
This PR is a fix for the mentioed bug, where `Add Security Rule` does not take list of `source users` as the parameter.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/PAN-OS/Integrations/Panorama/Panorama.yml [hunk 1]
```diff
@@ -5027,12 +5027,12 @@ script:
       - Yes
       - No
     - defaultValue: any
-      description: A comma-separated list of application object names for the rule. to create.
+      description: A comma-separated list of application object names for the rule to create.
       isArray: true
       name: application
-    - defaultValue: any
-      description: The source user for the rule to create.
+    - description: A comma-separated list of source users for the rule to create.
```
Comment (ShacharKidor) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.yml:5033]: According to the yml - did we change it in more than one command?

**Review by ShacharKidor — APPROVED** [review]
Very nice! 

@samuelFain  - Can we test it in the TPB? or in a UT?

File: Packs/PAN-OS/Integrations/Panorama/Panorama.yml (hunk 1 above)
Comment (samuelFain) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.yml:5033]: yes, its the same command - one for the pan-os mode and one for the Panorama mode of the integration.

Comment (ShacharKidor) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.yml:5033]: Yes, but worth changing it to test multiple source users, or alternatively add a unit test for that logic.

Comment (samuelFain) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.yml:5033]: @ShacharKidor Added a UT.

File: Packs/PAN-OS/ReleaseNotes/1_17_6.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#### Integrations
+
+##### Palo Alto Networks PAN-OS
+
+- Updated the *pan-os-create-rule* and *panorama-create-rule* commands to support a comma-separated list of values for their **source_user** argument.
```
Comment (GuyAfik) [inline Packs/PAN-OS/ReleaseNotes/1_17_6.md:5]:
```suggestion
- Updated the ***pan-os-create-rule*** command to support a comma-separated list of values for the *source_user* argument.
```

***panorama-create-rule*** is deprecated, no need to mention it in the RN.

Comment (samuelFain) [reply Packs/PAN-OS/ReleaseNotes/1_17_6.md:5]: Oh that makes sense now, thats why only _**pan-os-create-rule**_ is in the README.md

**Pull request merged**